clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/StorageService.java

```diff
@@ -88,6 +89,7 @@ public class StorageService extends AbstractVeniceService {
       boolean restoreDataPartitions,
       boolean restoreMetadataPartitions,
       Function<String, Boolean> checkWhetherStorageEngineShouldBeKeptOrNot,
+      Function<AbstractStorageEngine, Void> checkWhetherStoragePartitionsShouldBeKeptOrNot,
```

Comment:
Can we create an additional constructor that doesn't have `checkWhetherStoragePartitionsShouldBeKeptOrNot`, so if users don't want to use this, they wouldn't have to pass in a functional interface that does nothing.

Reply:
Ok, that sounds correct; `checkWhetherStoragePartitionsShouldBeKeptOrNot` is mainly being written in VeniceServer.